### ISSUE TITLE
Trim bundle size (and/or remove unused assets)

### ISSUE THREAD

**Issue by housseindjirdeh** [djirdehh/crypto_vue#3]
So the service worker file generated is a nice chunky 5MB

![image](https://user-images.githubusercontent.com/12476932/28592548-5bf5be48-7157-11e7-8418-b74585db09a0.png)

1. If there's room for improvement, will love to explore how we can cut our bundle sizes down to the users
2. All the images stored in the `static` folder, are all needed or can we remove some that are unused?

💃 🕺

**Comment by djirdehh** [comment]
Yeah I noticed this as well. I've compressed all the image assets already. I've added more images of cryptocurrency logos to the `static` folder than the current best ten performing purely to allow a visual display of new cryptocurrencies that may enter the top ten.

Which leaves me to consider eventually hosting all of the assets on an external server as opposed to storing them locally - since I'll be adding even more assets.

**Comment by housseindjirdeh** [comment]
Agreed it might definitely make more sense to host them on a server (if there isn't hosting that does it already)
